cob_bringup/robots/cob4-5.xml

```diff
@@ -131,6 +132,7 @@
 			<arg name="component_name" value="arm_left"/>
 			<arg name="can_device" value="can3"/>
 			<arg name="sim" value="$(arg sim)"/>
+			<arg name="cartesian_control" value="$(arg cartesian_control)"/>
```

Comment:
if we add this, then we add it for both arms...
however, adding it currently gives us load problems on the real hardware...
you could pass the argument only in case sim is active:
`<arg if=$(arg sim) name="cartesian_control" value="$(arg cartesian_control)"/>`